query/s3/s3_public_access_block_account.sql

```diff
@@ -0,0 +1,23 @@
+select
```

Comment:
@khushboo9024 Would we be able to make the query a bit more robust, like the `s3_public_access_block_bucket_account.sq` query (in particular when all settings are not blocked)?